pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -227,7 +227,7 @@ protected String getQuery(SlingHttpServletRequest request) {
 
     @Override
     protected Map<String, Object> resourceToMap(Resource resource) {
-        Module module = resource.adaptTo(Module.class);
+        com.redhat.pantheon.model.module.Module module = resource.adaptTo(com.redhat.pantheon.model.module.Module.class);
```

Comment:
Do we really need make this code change?

Reply:
Thanks @xdavidson  and @carlosmunoz.  I changed to use Java 8 and reverted the change to this file.

I'm happy to make any other changes. Thanks again for the help.